src/classes/music_album.rb

```diff
@@ -0,0 +1,13 @@
+class MusicAlbum
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify)
```

Comment:
As requirements say kindly make an association with the Item class and inherit its properties.